scipy/spatial/transform/rotation.py

```diff
@@ -1463,6 +1465,88 @@ def magnitude(self):
         else:
             return angles
 
+    def reduce(self, left=None, right=None, return_indices=False):
+        """Reduce this rotation with the provided rotation groups.
+
+        Parameters
+        ----------
+        left : `Rotation` instance, optional
+            Object containing the left rotation(s). Default value (`None`)
+            corresponds to the identity rotation.
+        right : `Rotation` instance, optional
+            Object containing the right rotation(s). Default value (`None`)
+            corresponds to the identity rotation.
+        return_indices : bool
```

Comment:
Same policy with optional.

I suggest modifying the description --- "Whether to return indices of rotations from `left` and `right` used for reduction." (adjust as see fit).

Reply:
I have modified it.